lib/menu.rb

```diff
@@ -0,0 +1,22 @@
+class Menu
+  
+  attr_reader :list_of_dishes
+  
+  def initialize
+  	@list_of_dishes = { "lasagne" => 8, 
+  		                "pizza" => 14, 
+  		                "classic burger" => 11,
+  		                "cheese burger" => 12,
+  		                "beef burger" => 13,
+  		                "salad" => 4 }
+  end
+
+  def list
+  	@list_of_dishes.each { |name,price| puts "#{name}....€#{price}." }
+  end
+
+  def dish_name
+  	list_of_dishes.map { |key, value| key }
```

Comment:
Lint/UnusedBlockArgument: Unused block argument - `value`. If it's necessary, use `_` or `_value` as an argument name to indicate that it won't be used. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)